include/swift/Basic/SanitizerOptions.h

```diff
@@ -0,0 +1,35 @@
+//===----- SanitizerOptions.h - Helpers related to sanitizers --*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===---------------------------------------------------------------------===//
+
+#ifndef SWIFT_BASIC_SANITIZER_OPTIONS_H
+#define SWIFT_BASIC_SANITIZER_OPTIONS_H
+
+#include "llvm/ADT/Triple.h"
+#include "llvm/Option/Arg.h"
+
+namespace swift {
+class DiagnosticEngine;
+
+enum class SanitizerKind : unsigned {
+  None = 0,
+  Address,
+};
+
+/// \brief Parses a -sanitize= argument's values.
+///
+/// \param Diag If non null, the argument is used to diagnose invalid values.
+/// \return Returns a SanitizerKind.
+SanitizerKind parseSanitizerArgValues(const llvm::opt::Arg *A,
```

Comment:
It seems weird for swiftBasic to depend on llvmOption. Maybe this belongs in a higher library, or maybe you can just pass the value string.

Also, if this doesn't support more than one sanitizer, I would name it "…Value" rather than "…Values".